conda.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: pbtest
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}
+
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
```

Comment:
I am hesitant to add the version of pybind11 used at build time to the build number, as per earlier discussion @wjakob .